seacrowd/sea_datasets/mswc/mswc.py

```diff
@@ -0,0 +1,308 @@
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_opus/train/0000.parquet?download=true",
+                ],
+            },
+        },
+        "validation": {
+            "cnh": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_wav/validation/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_opus/validation/0000.parquet?download=true",
+                ],
+            },
+            "ind": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_wav/validation/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_opus/validation/0000.parquet?download=true",
+                ],
+            },
+            "vie": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_wav/validation/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_opus/validation/0000.parquet?download=true",
+                ],
+            },
+        },
+        "test": {
+            "cnh": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_wav/test/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/cnh_opus/test/0000.parquet?download=true",
+                ],
+            },
+            "ind": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_wav/test/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/id_opus/test/0000.parquet?download=true",
+                ],
+            },
+            "vie": {
+                "wav": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_wav/test/0000.parquet?download=true",
+                ],
+                "opus": [
+                    "https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/vi_opus/test/0000.parquet?download=true",
+                ],
+            },
+        },
+    },
+}
```

Comment:
we only have 1 file per each split, lang, and format, rite? this is too complex to read, can we change to this f-string?
```suggestion
_URLS = f"https://huggingface.co/datasets/MLCommons/ml_spoken_words/resolve/refs%2Fconvert%2Fparquet/{lang}_{format}/{split}/0000.parquet?download=true"
```